flixel/input/gamepad/buttons/XBox360ButtonID.hx

```diff
@@ -1,11 +1,11 @@
-package flixel.input.gamepad;
+package flixel.input.gamepad.buttons;
 
 import flixel.input.gamepad.FlxGamepad;
 
 /**
  * Button IDs for Xbox 360 controllers
  */
-class XboxButtonID
+class XBox360ButtonID
```

Comment:
Technically, these classes having `Button` in the name is incorrect since they also contain axis IDs (it's also longer). So, if you're not tired of find-and-replace yet... :D Due to the package change the API for these classes isn't backwards compatible anyway.